### PR TITLE
v3.0, dev/benchmark: add how to run tpcc

### PR DESCRIPTION
**Pull request by jackysp**
Signed-off-by: Shuaipeng Yu <jackysp@gmail.com>

### What is changed, added or deleted? 
Add `how to run tpcc`

### Checklist 

- [ ] Add a new file to `TOC.md`
- [ ] No Tab spaces anywhere 
- [ ] Leave a blank line both before and after a code block
- [ ] Keep the first level heading consistent with `title` in metadata
- [ ] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

File: dev/benchmark/how-to-run-tpcc.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+ant
+```
+
+## 部署 TiDB 集群
+
+对于 1000 WAREHOUSE 我们将在 3 台服务器上部署集群。
+
+在 3 台服务器的条件下，建议每台机器部署 1 个 TiDB，1 个 PD 和 1 个 TiKV 实例。
+
+机器硬件配置:
```
Comment (lysu) [inline dev/benchmark/how-to-run-tpcc.md]:
```suggestion
比如这里采用的机器硬件配置是:
```

File: dev/benchmark/how-to-run-tpcc.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+| 类别 | 名称 |
+| :-: | :-: |
+| OS | Linux (CentOS 7.3.1611) |
+| CPU | 40 vCPUs, Intel(R) Xeon(R) CPU E5-2630 v4 @ 2.20GHz |
+| RAM | 128GB |
+| DISK | Optane 500GB SSD |
+
+对于采用了 NUMA 架构的服务器建议先用 `taskset` 进行绑核，首先用 `lscpu` 查看 NUMA node，比如：
```
Comment (lysu) [inline dev/benchmark/how-to-run-tpcc.md]:
```suggestion
因为该型号 CPU 是 NUMA 架构, 建议先用 `taskset` 进行绑核，首先用 `lscpu` 查看 NUMA node，比如：
```

File: dev/benchmark/how-to-run-tpcc.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+
+```text
+NUMA node0 CPU(s):     0,2,4,6,8,10,12,14,16,18,20,22,24,26,28,30,32,34,36,38
+NUMA node1 CPU(s):     1,3,5,7,9,11,13,15,17,19,21,23,25,27,29,31,33,35,37,39
+```
+
+可以通过下面的命令来启动 TiDB：
```
Comment (lysu) [inline dev/benchmark/how-to-run-tpcc.md]:
```suggestion
之后可以通过下面的命令来启动 TiDB：
```

File: dev/benchmark/how-to-run-tpcc.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+### TiKV 配置
+
+开始可以使用基本的配置，压测运行后可以通过观察 Grafana 并参考 [TiKV 调优说明](../reference/performance/tune-tikv.md)进行调整。
+
+### BenchmarkSQL 配置
+
+修改 `benchmarksql/run/props.mysql`：
+
+```text
+conn=jdbc:mysql://{TIDB-HOST}:{TIDB-PORT}/tpcc?useSSL=false&useServerPrepStmts=true&useConfigs=maxPerformance&sessionVariables=tidb_batch_commit=1
```
Comment (lysu) [inline dev/benchmark/how-to-run-tpcc.md]:
```suggestion
conn=jdbc:mysql://{HAPROXY-HOST}:{HAPROXY-PORT}/tpcc?useSSL=false&useServerPrepStmts=true&useConfigs=maxPerformance&sessionVariables=tidb_batch_commit=1
```

**Comment by jackysp** [comment]
PTAL @lilin90

**Pull request merged**